src/test/eventSchemas/explorer/registryItemNode.test.ts

```diff
@@ -16,13 +16,24 @@ import {
     assertNodeListOnlyContainsErrorNode,
     assertNodeListOnlyContainsPlaceholderNode,
 } from '../../utilities/explorerNodeAssertions'
-import { MockSchemaClient, MockToolkitClientBuilder } from '../../shared/clients/mockClients'
 import { clearTestIconPaths, IconPath, setupTestIconPaths } from '../../shared/utilities/iconPathUtils'
 import { asyncGenerator } from '../../utilities/collectionUtils'
 import globals from '../../../shared/extensionGlobals'
+import * as sinon from 'sinon'
+
+function createSchemaClient(data?: { schemas?: Schemas.SchemaSummary[]; registries?: Schemas.RegistrySummary[] }) {
+    return {
+        regionCode: 'code',
+        async *listSchemas(registryName: string, version: string): AsyncIterableIterator<Schemas.SchemaSummary> {
+            yield* data?.schemas ?? []
+        },
+        async *listRegistries() {
+            yield* data?.registries ?? []
+        },
+    } as unknown as SchemaClient
```

Comment:
Usually better to use `sinon` or `mockito` for type-safety though it is kind of nice to just instantiate a stub with data and not worry about the methods being called.

There are ways to generalize this, though by that point you're essentially creating server-side stubs. Which I believe the SDKs are already working on.